### PR TITLE
Fix debug_traceTransaction

### PR DESCRIPTION
**Pull request by gumb0**
- Rename `StandardTrace::json()` to `multilineTrace()`, because it's not a valid json now, but json objects for each opcode trace concantenated as seaparate line each.
- Move styled json option into separate `styledJson()` method.
- Add `jsonValue()` method to return json without converting to string.

### CODE REVIEW

File: libethereum/Executive.h [hunk 1]
```diff
@@ -61,7 +61,9 @@ class StandardTrace
     void setShowMnemonics() { m_showMnemonics = true; }
     void setOptions(DebugOptions _options) { m_options = _options; }
 
-    std::string json(bool _styled = false) const;
+    Json::Value jsonValue() const { return m_trace; }
```
Comment (chfast) [inline libethereum/Executive.h]: I would shorten it to `.json()` but I understand that refactoring was easier this way.

Comment (gumb0) [reply libethereum/Executive.h]: Yeah I considered it, then decided `jsonValue` is more explicit here, to show that it's not a string

Comment (chfast) [reply libethereum/Executive.h]: :+1:

**Comment by gumb0** [comment]
docker-aleth on circleci now takes longer than AppVeyor...

**Comment by chfast** [comment]
Yes. Sorry for that. It should not be running every build. I was about to fix that.

**Pull request merged**